docs/guides/main.rst

```diff
@@ -290,9 +290,19 @@ You can always get help using the ``beet help`` command. The plain ``beet help``
 command lists all the available commands; then, for example, ``beet help
 import`` gives more specific help about the ``import`` command.
 
+Core Concepts
+-------------
+
+* *Library*: Is the beets database model only meant to catalog users music , which means that it only represents links to music files. The library is commposed by @items(songs) and @albums. The metadata album is shared by the track groups. Each item belongs to an album, otherwise is a @singleton. Users can import songs with the ``beet import`` command. 
```

Comment:
The sentence about sharing metadata isn't very clear IMO (I'm not sure I understood your intention with that sentence correctly, the following points might be misguided):
* I guess you meant *album metadata* instead of  *metadata album*?
* The term *track groups* is not explained, something along the lines *items it contains* would stick to the expression introduced in the preceding sentence.